src/annotator/anchoring/xpath.ts

```diff
@@ -156,12 +141,12 @@ function evaluateSimpleXPath(xpath, root) {
  *
  * Example:
  *   node = nodeFromXPath('/main/article[1]/p[3]', document.body)
- *
- * @param {string} xpath
- * @param {Element} [root]
- * @return {Node|null}
  */
-export function nodeFromXPath(xpath, root = document.body) {
+export function nodeFromXPath(
+  xpath: string,
+  /* istanbul ignore next */
+  root: Element = document.body
```

Comment:
The default value is dropping the code coverage because this is a multi-line function now.

I've seen the `/* istanbul ignore next */` pattern used in othwr places, but I can try to add an extra test where `root` is not provided if you think it's better.

Reply:
> I've seen the /* istanbul ignore next */ pattern used in othwr places,

Using "ignore next" is fine here. The risk of not testing the default is minimal.